src/c/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME openvino_genai_c)
+
+file(GLOB SOURCE_FILES "${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp" "${CMAKE_CURRENT_SOURCE_DIR}/include/*.hpp")
+
+# shared library
+
+add_library(${TARGET_NAME} SHARED ${SOURCE_FILES})
+add_library(openvino::genai::c ALIAS ${TARGET_NAME})
+
+target_include_directories(${TARGET_NAME} PUBLIC "$<INSTALL_INTERFACE:runtime/include>"
+                                                 "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+                                                 $<BUILD_INTERFACE:$<TARGET_PROPERTY:openvino::genai,INTERFACE_INCLUDE_DIRECTORIES>>)
```

Comment:
Do we need add `""` as other target_include_directories? I think it is optional comments.
`"$<BUILD_INTERFACE:$<TARGET_PROPERTY:openvino::genai,INTERFACE_INCLUDE_DIRECTORIES>>")`

Reply:
includes dirs within `openvino::genai` are already wrapped with ""